### ISSUE TITLE
"

### ISSUE THREAD

**Issue by BrucePerens** [crystal-lang/crystal#11251]
The "```
In foo.cr:191:x

 191 | nation = (d.country || "USA"),
       ^
Error: Multiple assignment count mismatch
```

**Comment by asterite** [comment]
Agreed! Especially if we can detect that a newline comes after the comma. This error is pretty common.

**Comment by vlazar** [comment]
Can we use some synonym for `Spurious` please? (a request from a non native speaker 🙂)

**Comment by BrucePerens** [comment]
> Can we use some synonym for `Spurious` please? (a request from a non native speaker slightly_smiling_face)

Sure, how about `Superfluous` :smirk: 

I think `extra` would work.

**Comment by Fryguy** [comment]
Does it make more sense to point to the comma like below?  I'm not sure what the other non-spurious-comma form would look like for comparison.

```
In foo.cr:191:x

 191 | nation = (d.country || "USA"),
                                    ^
Error: Multiple assignment count mismatch
```

**Comment by BrucePerens** [comment]
Jason, I think that's expecting a bit more of the programmer than we should. It's sort of hard to connect multiple assignment with the comma unless you are already something of a language pro.

A non-spurious comma would look like this:
`a, b = 1, 2`.

**Comment by vlazar** [comment]
> Sure, how about `Superfluous` 😏

Supercalifragilisticexpialidocious would do

**Comment by Fryguy** [comment]
> It's sort of hard to connect multiple assignment with the comma unless you are already something of a language pro.

Agreed.  In my edited text there I meant to also change the error message to yours (now edited)

**Comment by BrucePerens** [comment]
Did this get a patch? I saw this today using compiler version 1.2.1:
```
In filename:
 22 | end
         ^
Error: invalid trailing comma in call
```
One problem, this one reports the line number *after* the trailing comma.

**Comment by asterite** [comment]
No, that's unrelated.
